packages/@aws-cdk/aws-kms/README.md

```diff
@@ -55,6 +55,22 @@ Note that a call to `.addToPolicy(statement)` on `myKeyImported` will not have
 an affect on the key's policy because it is not owned by your stack. The call
 will be a no-op.
 
+If a Key has an associated Alias, the Alias can be imported by name and used in place
+of the Key as a reference.
+
+```ts
+const myKeyAlias = kms.Alias.fromAliasName(this, 'myKey', 'alias/myKeyAlias');
+const bucket = new Bucket(this, 'MyEncryptedBucket', {
+    bucketName: 'myEncryptedBucket',
+    encryption: BucketEncryption.KMS,
+    encryptionKey: myKeyAlias
+});
```

Comment:
We should add an example to the ReadMe of how to refer to an AWS-managed account master key (that's the main motivation behind this feature).